recipes-kernel/linux/linux-tegra_4.4.bb

```diff
@@ -22,7 +22,7 @@ SRC_URI = "git://${KERNEL_REPO};branch=${SRCBRANCH} \
 "
 S = "${WORKDIR}/git"
 
-KERNEL_ROOTSPEC ?= "root=/dev/mmcblk\${devnum}p1 rw rootwait"
+KERNEL_ROOTSPEC ?= "root=/dev/mmcblk\${devnum}p${distro_bootpart} rw rootwait"
```

Comment:
I think you need a backslash before `${distro_bootpart}` here.

Reply:
Thanks, still working on testing these changes, will post a comment when done.

Reply:
Thanks for catching this, fixed and tested on hardware.